app/javascript/components/pages/NotFound.test.js

```diff
@@ -1,22 +1,19 @@
-// Imports React into our test file.
-import React from 'react'
 
-// Imports Enzyme testing and deconstructs Shallow into our test file.
+import React from 'react'
 import Enzyme, { shallow } from 'enzyme'
-
-// Imports Adapter utilizing the latest react version into our test file so we can run a testing render on any component we may need.
 import Adapter from 'enzyme-adapter-react-16'
-
-// Imports in the component we are going to be testing.
 import NotFound from './NotFound'
-
-//Allows us to utilize the adapter we import in earlier, allowing us to call and render a component.
 Enzyme.configure({ adapter: new Adapter() })
 
 describe("When NotFound renders", () => {
   it("displays a heading", () => {
     const notFound = shallow(<NotFound />)
-    const notFoundHeading = notFound.find("h3")
-    expect(notFoundHeading.text()).toEqual("This should fail")
+    const foundHeading = notFound.find("h1")
+    const notFoundHeading = notFound.find("h3")     
+    expect(foundHeading.length).toEqual(1)
+    expect(notFoundHeading.text()).toEqual("Not Found")
   })
-})
\ No newline at end of file
+})
+export default NotFound
+ 
```

Comment:
just delete line 17 - you don't need to export default in a test file

Reply:
bye bye line 17